docs/changelog/935.md

```diff
@@ -0,0 +1 @@
+- Improved documenation of [`action:scale-by-computed-dt-ratio`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-computed-dt-ratio), [`action:scale-by-computed-dt-part-ratio`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-computed-dt-part-ratio) and [`action:scale-by-dt`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-dt) with respect to time step size vs. time window. size.
```

Comment:
```suggestion
- Improved documentation of [`action:scale-by-computed-dt-ratio`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-computed-dt-ratio), [`action:scale-by-computed-dt-part-ratio`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-computed-dt-part-ratio) and [`action:scale-by-dt`](https://www.precice.org/configuration-xml-reference.html#actionscale-by-dt) with respect to time step size vs. time window size.
```

How does this PR improve the documentation? It only renames variables in preCICE.

Reply:
I think my rationale was that before the meaning of `dt` was often unclear (window or step?). This PR tries to make this clear.

Reply:
Probably it's the best solution to just not add anything to the changelog.